DNN Platform/Library/Services/UserProfile/UserProfilePicHandler.cs

```diff
@@ -163,16 +163,7 @@ public void ProcessRequest(HttpContext context)
             context.Response.Cache.SetExpires(DateTime.Now.AddMinutes(1));
             context.Response.Cache.SetMaxAge(new TimeSpan(0, 1, 0));
             context.Response.AddHeader("Last-Modified", DateTime.Now.ToString("r"));
-
-	        try
-	        {
-				context.Response.End();
-	        }
-			catch (ThreadAbortException)//if ThreadAbortException will shown, should catch it and do nothing.
-			{
-
-			}
-            
+            context.ApplicationInstance.CompleteRequest();
```

Comment:
Not really, it was throwing an exception on each request and annoyed me while debugging the telerik changes. Figured it could be bundled in since the change is pretty minor.